src/loader/endpoints/get_inloco_cities.py

```diff
@@ -1,51 +1,169 @@
 import pandas as pd
+import sys
+import numpy as np
+
+sys.path.append("..")
+sys.path.append("../src")
 from utils import secrets
 from endpoints.helpers import allow_local
 import endpoints.GoogleDocsLib as gd
-import endpoints.corrections as cr
+import Levenshtein as lev
 import os
+import yaml
 
 configs_path = os.path.join(os.path.dirname(__file__), "aux")
 cities_table = pd.read_csv(os.path.join(configs_path, "cities_table.csv"))
 states_table = pd.read_csv(os.path.join(configs_path, "states_table.csv"))
+corrections_table = yaml.load(
+    open(os.path.join(configs_path, "city_corrections.yaml"), "r"),
+    Loader=yaml.FullLoader,
+)
+
+
+class state_frame:
```

Comment:
Classes should always start with a capital letter, this case:

`StateFrame`

or `State_frame`